PWGHF/D2H/Tasks/CMakeLists.txt

```diff
@@ -58,3 +58,8 @@ o2physics_add_dpl_workflow(task-xicc
                     SOURCES taskXicc.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(task-sigmac
+                    SOURCES taskSigmac.cxx
+                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2::DetectorsVertexing
```

Comment:
O2::DetectorsVertexing should be removed I believe

Reply:
Removed